src/Envs/env_wrapper.jl

```diff
@@ -65,4 +59,4 @@ function testmode!(env::EnvWrapper, val::Bool=true)
 end
 
 trainable(env::EnvWrapper) = env.train
-game_over(env::EnvWrapper) = env.done
\ No newline at end of file
+game_over(env::EnvWrapper) = env.done
```

Comment:
Formatting error

Reply:
Oh, funny. I didn't realize I had touched that line. I had used the web editor. If this is a problem, I think you have permission to fix this inline.